### PR TITLE
MET-94: Adds docs about measurement limits

### PR DESCRIPTION
**Pull request by tseho**
**Description**

Screenshot_2020-04-02 How to Configure Measurement Limits — Akeneo PIM documentation

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

### CODE REVIEW

**Comment by juliensnz** [comment]
use env vars? what is the way of doing this @jjanvier ?

**Comment by tseho** [comment]
The comment in `pim_parameters.yml` make sense but requires more knowledge on the symfony 4.4 config, see https://github.com/akeneo/pim-community-dev/blob/e10d203597bcbba41595a58c8bcb7bcbc33a724d/config/services/pim_parameters.yml#L1-L2

File: manipulate_pim_data/measurement/configure_measurement_limits.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+How to Configure Measurement Limits
+===================================
+
+In the Measurement Families, some limits about the number of families and units have been defined to guarantee that the PIM is functional and runs smoothly.
+However, as your catalog is unique, **you may need to raise those values**, this chapter explains how.
+
+.. warning::
+    If you hit those limits, this could reflect that **your modelization is not adapted**. If it's the case, we can't guarantee that the PIM will be functional and runs smoothly, be sure to test the performances with your new limits.
+
+All those limits are defined as parameters that you can override like any other `Symfony config parameter <https://symfony.com/doc/3.4/best_practices/configuration.html>`_.
```
Comment (BitOne) [inline manipulate_pim_data/measurement/configure_measurement_limits.rst:10]: Wrong link, we are in SF 4.4

**Comment by BitOne** [comment]
Do you know which limit has more chance to get changed by a customer?

Because the problem with the solution you choose (with pim_parameter.yml), is it cannot be changed on the Serenity edition.

The Serenitiy edition uses env var to change this kind of parameter (see this configuration for example: https://github.com/akeneo/pim-community-dev/blob/e10d203597bcbba41595a58c8bcb7bcbc33a724d/config/services/pim_parameters.yml#L34) with the default value to be put in .env.

But if there are a lot of env var, it becomes very complex to manage. So we should keep it only for value that are at high risk to be changed.

File: manipulate_pim_data/measurement/configure_measurement_limits.rst (hunk 1 above)
Comment (tseho) [reply manipulate_pim_data/measurement/configure_measurement_limits.rst:10]: You are right but this page does not exists anymore in 4.4 with all the changes on parameters. 
The closest we have in the docs is only targeted at applications, not changing parameters of bundles.
see https://symfony.com/doc/4.4/best_practices.html#use-parameters-for-application-configuration

**Comment by tseho** [comment]
The study done by Delphine showed that most installations were only using the default families:
23/100 families
19/50 units on the largest family
3/5 operations on fahrenheit

Regarding the customers with customization on measurements, there were still under the limits that have been chosen.

There is only one known customer with more than 50 unit in one family but it's a badly designed family where they mixed all their unrelated metrics without any conversion.

We don't expect any need to changes this limits.

**Comment by BitOne** [comment]
Okay, thanks @tseho!

pim_parameters.yml then ;)

File: manipulate_pim_data/measurement/configure_measurement_limits.rst (hunk 1 above)
Comment (tseho) [reply manipulate_pim_data/measurement/configure_measurement_limits.rst]: fixed

**Comment by StevenVAIDIE** [comment]
Maybe we need to add minimum version impacted by this configuration in documentation ?

**Comment by tseho** [comment]
That's kinda implied by the documentation version. 
As you can see, I'm targeting master of the `pim-docs`, which is the documentation for the master branch of the PIM.
When we will release a new PIM version, let's say 5.0, the documentation will also have a corresponding release and since you can switch between versions in the header, you always have the up-to-date documentation to the version you are using.

**Comment by tseho** [comment]
What do you thing about the `requirejs.yml` @juliensnz  ?

**Pull request merged**